programs/openbook_v2/tests/utils.rs

```diff
@@ -0,0 +1,60 @@
+use std::ffi::OsStr;
+use solana_program::instruction::Instruction;
+use solana_program::pubkey::Pubkey;
+use solana_program_test::BanksClient;
+use solana_sdk::{
+    signature::Keypair, signer::Signer,
+};
+use solana_sdk::transaction::Transaction;
+
+// trying to find direction to protocol-v2
+pub fn get_paths() -> String {
```

Comment:
i wouldn't use this, just use target/deploy as the SBF/BPF_OUT_DIR and avoid all of this recursive path searching stuff, i'd just load the programs using `ProgramTest::add_program` & avoid this

Reply:
and openbook_v2.so needs to be where - i put in the tests/fixtures? or how will be added? here i am not quite sure.

Reply:
i was confused where to put tests/fixtures ..., now it is done in this way and removed unnecessary stuff